numeric-keyboard/src/main/java/com/konaire/numerickeyboard/NumericKeyboard.kt

```diff
@@ -174,5 +199,15 @@ class NumericKeyboard: FrameLayout {
 
             false
         }
+
+        keySpecial.text = keySpecialText
+        if (keySpecialText != "") {
+            keySpecial.isEnabled = true
```

Comment:
```suggestion
            keySpecial.isEnabled = keySpecialText.isNotEmpty()
```

and remove if of course